{
  "_annotation": {
    "cve_id": "CVE-2024-36404",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-36404.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:geotools:geotools:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "E4212152-5CE8-5D67-95CB-0A3B8213ED57",
                "versionEndExcluding": "29.6",
                "vulnerable": true
              },
              {
                "criteria": "cpe:2.3:a:geotools:geotools:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "AD2A0AD1-CCA9-5B92-BC09-FD34CA728B45",
                "versionEndExcluding": "30.4",
                "versionStartIncluding": "30.0",
                "vulnerable": true
              },
              {
                "criteria": "cpe:2.3:a:geotools:geotools:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "D4596401-1A1C-5873-91A4-B775119A097B",
                "versionEndExcluding": "31.2",
                "versionStartIncluding": "31.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}